src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -431,17 +431,18 @@ private void AIThink(Random random, in Entity entity, ref MicrobeAI ai, ref Heal
             }
         }
 
+        var isIronEater = organelles.IronBreakdownEfficiency > 0;
```

Comment:
I think there should be an added if here, that if experimental features are off then it forces `isIronEater` to false to disable all the new behaviours when siderohpore cannot be used.